sqldelight-gradle-plugin/src/test/fixtures/well-formed-selects/expected/com/sample/HockeyPlayerModel.java

```diff
@@ -185,6 +185,50 @@
   @NonNull
   HockeyPlayer.Position position();
 
+  interface Select_allModel {
+    HockeyPlayerModel hockey_player();
+
+    TeamModel team();
+  }
+
+  interface For_teamModel {
+    HockeyPlayerModel hockey_player();
+
+    TeamModel team();
+  }
+
+  interface Subquery_joinModel {
+    long _id();
+
+    int age();
+  }
+
+  interface Select_expressionModel {
+    String first_name();
+
+    long count();
+  }
+
+  interface Expression_subqueryModel {
+    HockeyPlayerModel hockey_player();
+
+    long size();
+  }
+
+  interface Some_joinModel {
+    HockeyPlayerModel hockey_player();
+
+    TeamModel team();
+  }
+
+  interface With_queryModel {
```

Comment:
With the forthcoming factory, we definitely should switch query name conventions to camelCase for the benefit of these interface names as well.
```
withQuery:
...
```
```java
public static SQLiteStatement withQuery() {
  //...
}
interface WithQueryModel { // <-- simple uppercasing of first letter only
}
```